src/Resource/Realms.php

```diff
@@ -99,6 +100,20 @@ public function adminEvents(string $realm, ?Criteria $criteria = null): array
             )
         );
     }
+    
+    public function keys(string $realm, ?Criteria $criteria = null): KeysMetadata
```

Comment:
To easily see whether this endpoint works all Keycloak versions, it's best if we have an integration test for this.

Some endpoints might only be available on older/newer Keycloak versions and it might happen that the representation classes have different properties.

I suggest extending the extending `RealmsTest`.